src/main/java/com/bootcamp/demo/data_access/AccountRepository.java

```diff
@@ -113,4 +114,13 @@ public String updatePassword(String username, String oldPassword, String newPass
                 .map(element-> element.toObject(Account.class))
                 .collect(Collectors.toList());
     }
+
+    public void addPromotionToAccount(String accountId, Promotion promotion) throws ExecutionException, InterruptedException, IllegalArgumentException {
+        DocumentReference accountReference = FirestoreClient.getFirestore().collection("accounts").document(accountId);
```

Comment:
Let's use a constant for collection name and reuse it everywhere. In case it will change we have to update in a single place